### PR TITLE
Fixes #2506 - Extract autogrow feature from bugform and apply to comment

### LINKED ISSUE #2506: auto-expand feature should be working on comments
When adding text to a comment the auto-expand doesn't work.

@Regaddi Do you want to work on this?

### PR DESCRIPTION
**Pull request by Regaddi**
This PR fixes issue #2506 

## Proposed PR background

Please provide enough information so that others can review your pull request:
This change will add auto-expand functionality on the new comment textfield on an issue details page.

---

- [x] I have read the [Pull Request + Code Style Guidelines](https://github.com/webcompat/webcompat.com/blob/master/docs/pr-coding-guidelines.md) thoroughly.

### CODE REVIEW

**Comment by miketaylr** [comment]
> Travis tests have failed

Blah, I need to figure out #2375...

**Comment by laghee** [comment]
Is the IO error after the `sleep 2` script significant at all? (It's the same one that was blocking the windows install, but I'm not sure why it would be intermittent.)

screenshot 2018-06-21 09 32 35

**Comment by miketaylr** [comment]
oooo, crud. I need to make that directory if it doesn't already exist. Thanks @laghee!

File: webcompat/static/js/lib/autogrow-textfield.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/*eslint new-cap: ["error", { "capIsNewExceptions": ["Deferred"] }]*/
```
Comment (miketaylr) [inline webcompat/static/js/lib/autogrow-textfield.js]: We don't need this exception for `Deferred` in this file, right?

Comment (miketaylr) [reply webcompat/static/js/lib/autogrow-textfield.js]: (not a big deal, if you want to push a fixup commit then merge, go for it. ^__^)

**Pull request merged**